Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataPatientBean.java

```diff
@@ -237,25 +215,29 @@ public boolean saveAddress() {
 
     public void editAddress() {
         if (selectedAddress != null) {
-            selectedAddress = loadTestDataService.getAddressBy(selectedAddress.getAddressId());
-            updateBeanWithAddress(selectedAddress);
+            withAddress = loadTestDataService.getAddressBy(selectedAddress.getAddressId());
         } else {
             addPatientErrorMessages(msgForSelectEdit(ADDRESS));
         }
     }
 
     public void newAddress() {
-        clearInputsAddresses();
+        withAddress = null;
     }
 
     public List<Identifier> getIdentifiers() {
-        return loadTestDataService.getAllIdentiersBy(getPatientId());
+        identifierList = loadTestDataService.getAllIdentiersBy(getPatientId());
+        return identifierList;
     }
 
     public boolean deleteIdentifier() {
         boolean result = false;
         if (selectedIdentifier != null) {
-            result = loadTestDataService.deleteIdentifier(selectedIdentifier);
+            if (CollectionUtils.isNotEmpty(identifierList) && identifierList.size() > 1) {
```

Comment:
I understand why you have both CollectionUtils.isNotEmpty and identifierList.size()>1 check here. Can you put comments on the code that you cannot delete identifierList if you have only one value